### ISSUE TITLE
Pytest not installed with behresp-dev env

### ISSUE THREAD

**Issue by jdebacker** [PSLmodels/Behavioral-Responses#75]
The `pytest` package is not installed as part of the `behresp-dev` environment.  This was surprising, given that `pytest` is a dependency of `taxcalc`, which is a dependency of `behresp`, but maybe there's been a change in how conda handles such dependencies.  

Should `pytest` be added as a dependency?  Are any other packages being missed?

```
(base) jason.debacker@x86_64-apple-darwin13 Behavioral-Responses % conda env create
Collecting package metadata (repodata.json): done
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.8.3
  latest version: 4.8.4

Please update conda by running

    $ conda update -n base -c defaults conda



Downloading and Extracting Packages
bokeh-2.2.0          | 5.4 MB    | ##################################### | 100%
wheel-0.35.1         | 37 KB     | ##################################### | 100%
setuptools-49.6.0    | 747 KB    | ##################################### | 100%
sqlite-3.33.0        | 1.3 MB    | ##################################### | 100%
typing_extensions-3. | 28 KB     | ##################################### | 100%
olefile-0.46         | 33 KB     | ##################################### | 100%
pip-20.2.2           | 1.7 MB    | ##################################### | 100%
cryptography-3.0     | 565 KB    | ##################################### | 100%
fsspec-0.8.0         | 70 KB     | ##################################### | 100%
taxcalc-3.0.0        | 30.3 MB   | ##################################### | 100%
pandas-1.1.1         | 8.0 MB    | ##################################### | 100%
tbb-2020.2           | 155 KB    | ##################################### | 100%
ca-certificates-2020 | 124 KB    | ##################################### | 100%
cffi-1.14.2          | 220 KB    | ##################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate behresp-dev
#
# To deactivate an active environment, use
#
#     $ conda deactivate

(base) jason.debacker@x86_64-apple-darwin13 Behavioral-Responses % conda activate behresp-dev
(behresp-dev) jason.debacker@x86_64-apple-darwin13 Behavioral-Responses % ls
LICENSE.md		behresp			gitpr.bat
Makefile		behresp.egg-info	gitsync
PSL_catalog.json	conda.recipe		gitsync.bat
README.md		continuous_integration	pytest.ini
RELEASES.md		docs			setup.py
RELEASING.md		environment.yml
appveyor.yml		gitpr
(behresp-dev) jason.debacker@x86_64-apple-darwin13 Behavioral-Responses % cd behresp
(behresp-dev) jason.debacker@x86_64-apple-darwin13 behresp % pytest
zsh: command not found: pytest
(behresp-dev) jason.debacker@x86_64-apple-darwin13 behresp % conda list
# packages in environment at /Users/jason.debacker/anaconda3/envs/behresp-dev:
#
# Name                    Version                   Build  Channel
blas                      1.0                         mkl
bokeh                     2.2.0                    py38_0
brotlipy                  0.7.0           py38haf1e3a3_1000
ca-certificates           2020.7.22                     0
certifi                   2020.6.20                py38_0
cffi                      1.14.2           py38hed5b41f_0
chardet                   3.0.4                 py38_1003
cryptography              3.0              py38hddc9c9b_0
freetype                  2.10.2               ha233b18_0
fsspec                    0.8.0                      py_0
idna                      2.10                       py_0
intel-openmp              2019.4                      233
jinja2                    2.11.2                     py_0
jpeg                      9b                   he5867d9_2
lcms2                     2.11                 h92f6f08_0
libcxx                    10.0.0                        1
libedit                   3.1.20191231         h1de35cc_1
libffi                    3.3                  hb1e8313_2
libllvm9                  9.0.1                h21ff451_1
libpng                    1.6.37               ha441bb4_0
libtiff                   4.1.0                hcb84e12_1
llvm-openmp               10.0.0               h28b9765_0
llvmlite                  0.33.0           py38ha11be7d_1
lz4-c                     1.9.2                hb1e8313_1
markupsafe                1.1.1            py38h1de35cc_1
marshmallow               3.7.1                      py_0
mkl                       2019.4                      233
mkl-service               2.3.0            py38hfbe908c_0
mkl_fft                   1.1.0            py38hc64f4ea_0
mkl_random                1.1.1            py38h959d312_0
ncurses                   6.2                  h0a44026_1
numba                     0.50.1           py38h959d312_1
numpy                     1.19.1           py38h3b9f5b6_0
numpy-base                1.19.1           py38hcfb5961_0
olefile                   0.46                       py_0
openssl                   1.1.1g               h1de35cc_0
packaging                 20.4                       py_0
pandas                    1.1.1            py38hb1e8313_0
paramtools                0.15.0             pyh9f0ad1d_0    conda-forge
pillow                    7.2.0            py38ha54b6ba_0
pip                       20.2.2                   py38_0
pycparser                 2.20                       py_2
pyopenssl                 19.1.0                     py_1
pyparsing                 2.4.7                      py_0
pysocks                   1.7.1                    py38_1
python                    3.8.5                h26836e1_0
python-dateutil           2.8.1                      py_0
pytz                      2020.1                     py_0
pyyaml                    5.3.1            py38haf1e3a3_1
readline                  8.0                  h1de35cc_0
requests                  2.24.0                     py_0
setuptools                49.6.0                   py38_0
six                       1.15.0                     py_0
sqlite                    3.33.0               hffcf06c_0
taxcalc                   3.0.0                    py38_0    PSLmodels
tbb                       2020.2               h879752b_0
tk                        8.6.10               hb0a8c7a_0
tornado                   6.0.4            py38h1de35cc_1
typing_extensions         3.7.4.3                    py_0
urllib3                   1.25.10                    py_0
wheel                     0.35.1                     py_0
xz                        5.2.5                h1de35cc_0
yaml                      0.2.5                haf1e3a3_0
zlib                      1.2.11               h1de35cc_3
zstd                      1.4.5                h41d2c2f_0
```

**Comment by jdebacker** [comment]
PR #74 adds `pytest` to the `behresp-dev` environment.

**Comment by MattHJensen** [comment]
Closing as resolved by #74.
